src/tests/IntegrationTest.test.jsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] In this project we are not allowed to use eslint-disabler. At that point, you can fix the issue by removing unused imported items. Kindly remove, eslint-disabler from your code!